components/Users.js

```diff
@@ -0,0 +1,19 @@
+import Link from 'next/link'
+
+const Users = ({ userId, userName }) =>
+  (
+    <div className="flex flex-row p-3 my-4 shadow-md lg:w-1/3 md:w-1/2">
+      <div className="w-5/6">
+        <div>
+          {userName}
+        </div>
```

Comment:
Let's not get into the pitfall of `div`s. For this case, we can use `h1` / `h2` / `p` instead.